src/subscript/runrms/runrms.py

```diff
@@ -449,11 +500,11 @@ def _fsplitter(xline):  # yes... an inner function
             raise SystemExit
 
         try:
-            with open(pathlib.Path(self.project) / "project_lock_file", "r") as lockf:
-                for line in lockf.readlines():
-                    self.lockf = line.replace("\n", "")
-                    self.locked = True
-                    break
+            lockfiletxt = (pathlib.Path(self.project) / "project_lock_file").read_text()
```

Comment:
pylint will complain here about missing encoding?

Reply:
Did not notice that but have added encoding